practice/practice-02/src/styles/layouts/_welcome.scss

```diff
@@ -29,24 +30,32 @@
   }
 
   &__lines-background {
-    @include setAbsoluteAndLocation($absolute: true, $top: -76px, $right: 0);
+    @include set-absolute-and-location($absolute: true, $top: -76px, $right: 0);
     @include size(187px, 430px);
   }
 
   &__arrow-img-up {
-    @include setAbsoluteAndLocation($absolute: true, $top: -15px, $right: 50px);
+    @include set-absolute-and-location(
+      $absolute: true,
+      $top: -15px,
+      $right: 50px
+    );
 
     @include size(43px, 215px);
   }
 
   &__mint-img {
-    @include setAbsoluteAndLocation($absolute: true, $bottom: 100px, $right: 0);
+    @include set-absolute-and-location(
+      $absolute: true,
+      $bottom: 100px,
+      $right: 0
+    );
   }
   // End banner
 
   // Start content
   &__content {
-    @include flex-container($flex-direction: column);
+    @include flex-container(true, $flex-direction: column);
     @include size(calc(428px - 48px), auto);
```

Comment:
Please count the value instead of using this. If you want to count it related to variable, you can use calc(428px - $containerPadding)